lib/rules/no-at-ember-render-modifiers.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const ERROR_MESSAGE =
+  'Do not use @ember/render-modifiers. Instead, use derived data patterns, and/or co-locate destruction via @ember/destroyable';
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+/** @type {import('eslint').Rule.RuleModule} */
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'disallow importing from @ember/render-modifiers',
+      category: 'Miscellaneous',
```

Comment:
Would `Ember Octane` or `Deprecations` be a better category for this?

Reply:
updated!